common/source/docs/common-autopilots.rst

```diff
@@ -88,4 +92,101 @@ available if you're still working on those platforms:
    PX4FMU
    Qualcomm Snapdragon Flight Kit
 
+.. _common-autopilots_limited_firmware:
+
+Firmware Limitations on Selected Hardware
+-----------------------------------------
+
+The ArduPilot firmware in some configurations exceeds 1 MB in size. Some autopilots may not have enough
+flash memory to store the full firmware.
+
+For the affected autopilots, a minimal firmware is generated. This firmware omits less-commonly used features
```

Comment:
instead of "minimal" perhaps "reduced", since it really isnt the minimal set that could fly or operate

Reply:
Fixed